common-utils/src/common/deployment/__init__.py

```diff
@@ -179,31 +162,37 @@ class FlowDockerEnv(BaseModel):
     dockerfile_path: FilePath
     python_version: Literal["3.10"]
     docker_build_context: DirectoryPath = Path(".")
+    project_name: StrictStr
```

Comment:
we now need to provide the `FlowDockerEnv` model with project name, which is fine since the internals of the deployment module is the only process that creates these.